[WIP] OBSDOCS-63: Update info re supported Alertmanager configurations (#79966)
openshift/openshift-docs: modules/monitoring-configuring-notifications-for-user-defined-alerts.adoc

// * observability/monitoring/managing-alerts.adoc

:_mod-docs-content-type: PROCEDURE
[id="configuring-notifications-for-user-defined-alerts_{context}"]
= Configuring notifications for user-defined alerts

> If you have enabled a separate instance of Alertmanager dedicated to user-defined alert routing, you can customize where and how the instance sends notifications by overwriting its configuration. You can achieve this by editing the `alertmanager-user-workload` secret in the `openshift-user-workload-monitoring` namespace.

Reviewer: ```suggestion
After you enable a separate instance of Alertmanager that is dedicated to user-defined alert routing, you can customize where and how the instance sends notifications by editing the `alertmanager-user-workload` secret in the `openshift-user-workload-monitoring` namespace.
```
it's a long sentence, but more clear IMHO

Author: The "if you..." wording is better here IMO, because the user can choose not to do it this way (you don't have the enable the seperate instance of Alertmanager for user-defined projects, and can configure receivers in other ways).

Author: But I will definitely include the rest of the suggestions here, thank you!
